SU2_PY/SU2_FSI/AdjointInterface.py

```diff
@@ -0,0 +1,866 @@
+#!/usr/bin/env python
+
+## \file AdjointInterface.py
+#  \brief Interface class that handles adjoint FSI synchronisation and communication.
+#  \author Ruben Sanchez
+#  \version 7.0.0
+#
+# SU2 Original Developers: Dr. Francisco D. Palacios.
+#                          Dr. Thomas D. Economon.
+#
+# SU2 Developers: Prof. Juan J. Alonso's group at Stanford University.
+#                 Prof. Piero Colonna's group at Delft University of Technology.
+#                 Prof. Nicolas R. Gauger's group at Kaiserslautern University of Technology.
+#                 Prof. Alberto Guardone's group at Polytechnic University of Milan.
+#                 Prof. Rafael Palacios' group at Imperial College London.
+#                 Prof. Edwin van der Weide's group at the University of Twente.
+#                 Prof. Vincent Terrapon's group at the University of Liege.
+#
+# Copyright (C) 2012-2019 SU2, the open-source CFD code.
+#
+# SU2 is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
```

Comment:
This does not match what we have in other files also, 7.0.2 (same for other files)